<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 18pt  &quot;S-Core Dream&quot;;
color: rgb(217, 217, 217);
</string>
  </property>
  <widget class="QPushButton" name="start_signupbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(111, 11, 13);
color:white</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>270</y>
     <width>410</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font: 50pt &quot;a타임머신&quot;;</string>
   </property>
   <property name="text">
    <string>날씨우체통</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_loginbutton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(111, 11, 13);
color: white
</string>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image : url('background.jpg');</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>start_signupbutton</zorder>
  <zorder>label_2</zorder>
  <zorder>start_loginbutton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
